com.unity.netcode.gameobjects/Runtime/Spawning/NetworkSpawnManager.cs

```diff
@@ -382,6 +283,33 @@ internal void ChangeOwnership(NetworkObject networkObject, ulong clientId)
             }
         }
 
+        internal bool HasPrefab(bool isSceneObject, uint globalObjectIdHash)
+        {
+            if (!NetworkManager.NetworkConfig.EnableSceneManagement || !isSceneObject)
+            {
+                if (NetworkManager.PrefabHandler.ContainsHandler(globalObjectIdHash))
+                {
+                    return true;
+                }
+                if (NetworkManager.NetworkConfig.NetworkPrefabOverrideLinks.ContainsKey(globalObjectIdHash))
```

Comment:
Even i here i would suggest using TryGetValue as you are accessing the hash table twice.